src/index.js

```diff
@@ -28,26 +29,24 @@ export const actions = withHandlers({
   decrement: ({ setCounter }) => () => setCounter(n => n - 1),
 });
 
+export const counterStyle = Style.of({
+  flex: 1,
+  justifyContent: "center",
+  alignItems: "center",
+});
+
 export const Counter = (
   {
     increment,
     counter,
     decrement,
   }: IWithHandlers,
 ) => (
-  <View style={style.counter}>
+  <View style={counterStyle.resolve()}>
```

Comment:
Like this (with ramda)?

![screen shot 2017-04-02 at 4 38 15 pm](https://cloud.githubusercontent.com/assets/9259509/24590860/d2b7459a-17c2-11e7-8d4a-5554db6189c9.png)